packages/contracts/test/contracts/OVM/bridge/base/OVM_L1CrossDomainMessenger.spec.ts

```diff
@@ -89,15 +90,25 @@ describe('OVM_L1CrossDomainMessenger', () => {
 
   let OVM_L1CrossDomainMessenger: Contract
   beforeEach(async () => {
-    const xDomainMessenerImpl = await Factory__OVM_L1CrossDomainMessenger.deploy()
+    const xDomainMessengerImpl = await Factory__OVM_L1CrossDomainMessenger.deploy()
     // We use an upgradable proxy for the XDomainMessenger--deploy & set up the proxy.
     OVM_L1CrossDomainMessenger = await deployProxyXDomainMessenger(
       AddressManager,
-      xDomainMessenerImpl
+      xDomainMessengerImpl
     )
     await OVM_L1CrossDomainMessenger.initialize(AddressManager.address)
   })
 
+  describe('pause', () => {
```

Comment:
Can we add a quick `when called by account other than the owner` just for completion here? :)